draft-ietf-quic-transport.md

```diff
@@ -2206,9 +2212,17 @@ The endpoint MUST use unpredictable data in every PATH_CHALLENGE frame so that
 it can associate the peer's response with the corresponding PATH_CHALLENGE.
 
 An endpoint MUST expand datagrams that contain a PATH_CHALLENGE frame to at
-least the smallest allowed maximum datagram size of 1200 bytes.  Sending UDP
-datagrams of this size ensures that the network path from the endpoint to the
-peer can be used for QUIC; see {{datagram-size}}.
+least the smallest allowed maximum datagram size of 1200 bytes, unless the
+anti-amplification limit for the path does not permit sending a datagram of
+this size.  Sending UDP datagrams of this size ensures that the network path
+from the endpoint to the peer can be used for QUIC; see {{datagram-size}}.
+
+If an endpoint is unable to expand the datagram payload to 1200 bytes, no
+padding is necessary.  However, this means that the path MTU will not be
+validated.  The endpoint MUST perform a second path validation with a datagram
```

Comment:
Just a wording nit: From context it's clear that this is a qualified MUST, but perhaps we could include that in the sentence itself? Even something like "In this situation, the endpoint MUST" such that if someone were to quote this sentence it would be clear that there's more context that applies.